aten/src/ATen/templates/Functions.h

```diff
@@ -40,32 +41,49 @@ AT_FORALL_SCALAR_TYPES_AND3(Bool, Half, BFloat16, TENSOR)
 AT_FORALL_COMPLEX_TYPES(TENSOR)
 #undef TENSOR
 
-${function_declarations}
+${function_definitions}
 
 // Special C++ only overloads for std()-like functions (See gh-40287)
 // These are needed because int -> bool conversion takes precedence over int -> IntArrayRef
 // So, for example std(0) would select the std(unbiased=False) overload
-TORCH_API Tensor var(const Tensor& self, int dim);
-TORCH_API std::tuple<Tensor,Tensor> var_mean(const Tensor& self, int dim);
-TORCH_API Tensor std(const Tensor& self, int dim);
-TORCH_API std::tuple<Tensor,Tensor> std_mean(const Tensor& self, int dim);
+TORCH_API inline Tensor var(const Tensor& self, int dim) {
+  return at::var(self, IntArrayRef{dim});
+}
+TORCH_API inline std::tuple<Tensor, Tensor> var_mean(const Tensor& self, int dim) {
+  return at::var_mean(self, IntArrayRef{dim});
+}
+TORCH_API inline Tensor std(const Tensor& self, int dim) {
+  return at::std(self, IntArrayRef{dim});
+}
+TORCH_API inline std::tuple<Tensor, Tensor> std_mean(const Tensor& self, int dim) {
+  return at::std_mean(self, IntArrayRef{dim});
+}
 
 
 // Special C++ only overloads for convnd functions (See gh-45667)
 // These are needed because {1, 2} is ambiguous between string and IntArrayRef overloads
-TORCH_API at::Tensor conv1d(
+TORCH_API inline at::Tensor conv1d(
     const Tensor& input, const Tensor& weight, const Tensor& bias, IntArrayRef stride,
-    std::initializer_list<int64_t> padding, IntArrayRef dilation = 1, int64_t groups = 1);
-TORCH_API at::Tensor conv2d(
+    std::initializer_list<int64_t> padding_, IntArrayRef dilation = 1, int64_t groups = 1) {
+  auto padding = IntArrayRef(padding_);
```

Comment:
What's going on here? Initializer list should implicitly construct to IntArrayRef

Reply:
I copied that straight in from where it used to live in Functions.cpp. I didn't bother checking for whether there was an implicit conversion- cool, I'll fix up the other conv methods here too.

Reply:
We actually might be able to get rid of these; they were added in https://github.com/pytorch/pytorch/pull/45667/ because {1, 2} can implicitly be converted to a `std::string`, but that doesn't matter now that we've moved to `c10::string_view`. I'll confirm locally.

Reply:
> {1, 2} can implicitly be converted to a std::string

bruhhh LOL. OK, either way is fine then.